webservices/css/styling_guidelines.md

```diff
@@ -43,3 +71,77 @@ It's better to have these all defined in one place, where they can be easily ref
 #### Favor targeting elements rather than pages
 
 Something in suspenders appends some seemingly helpful classes to the `<body>` element that specify the controller and action, but they are a trap. Use separate descriptive class names instead of having a single class that is styled or positioned differently depending on what template was rendered.
+
+####Images as Background*
+When adding images to your design, especially if it's going to be responsive, use a <div> tag with the background CSS property instead of <img> elements.
+
+This may seem like more work for nothing, but it actually makes it much easier to style images properly, keeping their original size and aspect-ratio, thanks to background-size, background-position, and other properties.
+
+CSS / HTML
+```
+img {
+    width: 300px;
+    height: 200px;
+}
+
+div {
+    width: 300px;
+    height: 200px;
+    background: url('https://tutorialzine.com/media/2016/08/bicycle.jpg');
+    background-position: center center;
+    background-size: cover;
+}
+
+section{
+    float: left;
+    margin: 15px;
+}
+```
+
+A drawback of this technique is that the web accessibility of your page will take a slight hit, as images won't be crawled properly by screen readers and search engines. This issue can be resolved by the awesome object-fit but it doesn't have full browser support yet.
+
+
+####Don’t forget to comment:
+CSS might not be a programming language but its code still needs to be documented. Some simple comments are all it takes to organize a style sheet and make it more accessible to your colleagues or your future self.
+For larger sections of the CSS such as major components or media-queries, use a stylized comment and leave a couple of new lines after:
+```
+/*---------------
+    #Header
+---------------*/
+header { }
+
+header nav { }
+
+/*---------------
+    #Slideshow
+---------------*/
+.slideshow { }
+```
+
+Details in the design or less important components can be marked with a single-line comment.
+
+```
+/*   Footer Buttons   */
+.footer button { }
+
+.footer button:hover { }
+```
+
+Also, remember that CSS doesn't have single line  comments, so when commenting something out you still need to use the / / syntax.
```

Comment:
should this read "...so when commenting something out you still need to use the `/* */` syntax"?